src/Restaurant.js

```diff
@@ -1,14 +1,15 @@
 import React from "react";
 
+import Orders from "./Order";
```

Comment:
```suggestion
import Order from "./Order";
```
You need to double check this one - you're importing a singular `Order` here :)